frontend/src/assets/css/variables.scss

```diff
@@ -0,0 +1,7 @@
+:root {
+  --white-pale: #f8f9fa;
```

Comment:
It's totally white on design 
```suggestion
  --white: #ffffff;
```
<img width="450" alt="Screenshot 2023-08-20 at 19 28 13" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/64521658/514afc95-e375-413f-8fef-cb99627c0eea">